catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -579,8 +578,8 @@ class PoppySegmentedCommand(SegmentedAperture):
                                 [piston, tip, tilt, defocus, ...] (Noll convention)
     :param dm_config_id: str, name of the section in the config_ini file where information
                          regarding the segmented DM can be found.
-    :param laser_config_id: str, name of the section in the config_ini file where information
-                            regarding the laser can be found.
+    :param wavelength: wavelength: float, wavelength in nm of the poppy optical system used for
+                        (extremely oversimplified) focal plane simulations
```

Comment:
I am not sure it is necessary to say "extremely oversimplified", maybe just "simplified"?

Reply:
"Simplified" to me is when we put in a thin lens in a simulation to approximate an out-of focus optic, "extremely simplified" is to me when we literally ignore the *entire* optical setup and just do one single Fourier transform from the pupil to a focal plane, like in this case with the PSF display ^^

Reply:
Okay, but those words don't necessarily mean exactly the same thing to others. Maybe point to the poppy documentation so that people can understand that for themselves?